crates/teloxide-core/src/adaptors/trace.rs

```diff
@@ -66,30 +66,30 @@ bitflags::bitflags! {
     /// ```
     pub struct Settings: u8 {
         /// Trace requests (only request kind, e.g. `send_message`)
-        const TRACE_REQUESTS = 0b00000001;
+        const TRACE_REQUESTS = 1;
 
         /// Trace requests verbosely (with all parameters).
         ///
         /// Implies [`TRACE_REQUESTS`]
-        const TRACE_REQUESTS_VERBOSE = 0b00000011;
+        const TRACE_REQUESTS_VERBOSE = (1 << 1) | Self::TRACE_REQUESTS.bits();
 
         /// Trace responses (only request kind, e.g. `send_message`)
-        const TRACE_RESPONSES = 0b00000100;
+        const TRACE_RESPONSES = 1 << 2;
 
         /// Trace responses verbosely (with full response).
         ///
         /// Implies [`TRACE_RESPONSES`]
-        const TRACE_RESPONSES_VERBOSE = 0b00001100;
+        const TRACE_RESPONSES_VERBOSE = (1 << 3) | Self::TRACE_RESPONSES.bits();
 
         /// Trace everything.
         ///
         /// Implies [`TRACE_REQUESTS`] and [`TRACE_RESPONSES`].
-        const TRACE_EVERYTHING = Self::TRACE_REQUESTS.bits | Self::TRACE_RESPONSES.bits;
+        const TRACE_EVERYTHING = Self::TRACE_REQUESTS.bits() | Self::TRACE_RESPONSES.bits();
```

Comment:
Any reason to use `.bits()` instead of `.bits`?

Reply:
Seems like the guidance changed when switching versions (`1.*` suggest using `.bits`: https://docs.rs/bitflags/1.3.2/bitflags/macro.bitflags.html). I would prefer using the field here, simply because it's a bit shorter.